packages/web-components/@sfx/autocomplete/src/events.ts

```diff
@@ -0,0 +1 @@
+export const autocompleteReceivedResults = 'sfx::autocomplete_received_results';
```

Comment:
I would consider using SCREAMING_SNAKE_CASE for this since it's meant to be a constant.

```suggestion
export const AUTOCOMPLETE_RECEIVED_RESULTS = 'sfx::autocomplete_received_results';
```